doc/results.md

```diff
@@ -260,48 +259,50 @@ return      (_10010101 + i286) << 6 + _1111
 | _N_ | `ffffffffffffffffffffffffffffffffffffffffffffffffc7634d81f4372ddf581a0db248b0a77aecec196accc52973` |
 | _d_ | `2` |
 | _N_-_d_ | `ffffffffffffffffffffffffffffffffffffffffffffffffc7634d81f4372ddf581a0db248b0a77aecec196accc52971` |
-| Length | 434 |
-| Algorithm | `opt(dictionary(hybrid(4,0),continued_fractions(dichotomic)))` |
+| Length | 433 |
+| Algorithm | `opt(dictionary(hybrid(6,0,sliding_window_short(5,0)),continued_fractions(dichotomic)))` |
 | Best Known | 433 |
-| Delta | +1 |
+| Delta | +0 |
 
 
 Addition chain produced by `addchain`:
 
 ```go
-_10      = 2*1
-_11      = 1 + _10
-_101     = _10 + _11
-_111     = _10 + _101
-_1001    = _10 + _111
-_1011    = _10 + _1001
-_1101    = _10 + _1011
-_1111    = _10 + _1101
-_11110   = 2*_1111
-_11111   = 1 + _11110
-_1111100 = _11111 << 2
-i14      = _1111100 << 2
-i26      = (i14 << 3 + _1111100) << 7 + i14
-i42      = i26 << 15 + i26
-x64      = i42 << 30 + i42 + _1111
-x128     = x64 << 64 + x64
-x192     = x128 << 64 + x64
-x194     = x192 << 2 + _11
-i225     = ((x194 << 6 + _111) << 3 + _11) << 7
-i235     = 2*((_1101 + i225) << 6 + _1101) + 1
-i258     = ((i235 << 11 + _11111) << 2 + 1) << 8
-i269     = ((_1101 + i258) << 2 + _11) << 6 + _1011
-i286     = ((i269 << 4 + _111) << 6 + _11111) << 5
-i308     = ((_1011 + i286) << 10 + _1101) << 9 + _1101
-i323     = ((i308 << 4 + _1011) << 6 + _1001) << 3
-i340     = ((1 + i323) << 7 + _1011) << 7 + _101
-i357     = ((i340 << 5 + _111) << 5 + _1111) << 5
-i369     = ((_1011 + i357) << 4 + _1011) << 5 + _111
-i387     = ((i369 << 3 + _11) << 7 + _11) << 6
-i397     = ((_1011 + i387) << 4 + _101) << 3 + _11
-i413     = ((i397 << 4 + _11) << 4 + _11) << 6
-i427     = ((_101 + i413) << 5 + _101) << 6 + _1011
-return     (2*i427 + 1) << 4 + 1
+_10       = 2*1
+_11       = 1 + _10
+_101      = _10 + _11
+_111      = _10 + _101
+_1000     = 1 + _111
+_1001     = 1 + _1000
+_1011     = _10 + _1001
+_1101     = _10 + _1011
+_1111     = _10 + _1101
+_10111    = _1000 + _1111
+_11001    = _10 + _10111
+_11011    = _10 + _11001
+_11111    = _1000 + _10111
+_1111100  = _11111 << 2
+_11111000 = 2*_1111100
+i17       = 2*_11111000
+i23       = i17 << 5 + i17
+i34       = i23 << 10 + i23
+i61       = (i34 << 4 + _11111000) << 21 + i34
+i113      = (i61 << 3 + _1111100) << 47 + i61
```

Comment:
It is confusing as to why we're adding even numbers where the least significant bits don't contribute anything. I wonder if this indicates a bug in the new windowing algorithm where it doesn't realize that trailing zeros are worthless.